packages/devtools_server/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 0.2.6-dev.2
 - Updated package:vm_service to >= 4.1.0
+- Made `serveDevtools` Serve requests from in an error zone and log errors.
```

Comment:
You can remove this change. We put all changelog entries in devtools/CHANGELOG.md and we have a script that generates the entries when it's time to do the release.

Reply:
Thanks, will do!

Reply:
done.